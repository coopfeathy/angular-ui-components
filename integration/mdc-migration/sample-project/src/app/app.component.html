<autocomplete-example></autocomplete-example>
<button-example></button-example>
<card-example></card-example>
<chips-example></chips-example>
<form-field-example></form-field-example>
<input-example></input-example>
<list-example></list-example>
<menu-example></menu-example>
<paginator-example></paginator-example>
<progress-bar-example></progress-bar-example>
<progress-spinner-example></progress-spinner-example>
<radio-example></radio-example>
<select-example></select-example>
<slide-toggle-example></slide-toggle-example>
<slider-example></slider-example>
<snack-bar-example></snack-bar-example>
