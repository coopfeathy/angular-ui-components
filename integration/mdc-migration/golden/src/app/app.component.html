<chips-example></chips-example>
<menu-example></menu-example>
<paginator-example></paginator-example>
<progress-bar-example></progress-bar-example>
<progress-spinner-example></progress-spinner-example>
